channels/us.m3u

```diff
@@ -3470,50 +3456,20 @@ https://rds3.desdeparaguay.net/tvs/tvs/chunklist_w685773919.m3u8
 https://rpn1.bozztv.com/36bay2/gusa-hitops/tracks-v1a1/mono.m3u8
 #EXTINF:-1 tvg-id="TVSHollywoodHistory.us" tvg-name="TVS Hollywood History" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/pXOyNEy.png" group-title="Classic",TVS Hollywood History (408p)
 https://rpn1.bozztv.com/36bay2/gusa-tvshollywoohistory/tracks-v1a1/mono.m3u8
-#EXTINF:-1 tvg-id="TVSHorrorNetwork.us" tvg-name="TVS Horror Network" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/uNQeGHw.jpg" group-title="Classic",TVS Horror Network (480p)
-https://rpn1.bozztv.com/36bay2/gusa-tvshorror/tracks-v1a1/index-1623401253-now.m3u8
-#EXTINF:-1 tvg-id="TVSInspirationalNetwork.us" tvg-name="TVS Inspirational Network" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/pXOyNEy.png" group-title="Classic",TVS Inspirational Network (480p)
-https://rpn1.bozztv.com/36bay2/gusa-TVSInspirationalNetwork/tracks-v1a1/index-1623401418-now.m3u8
-#EXTINF:-1 tvg-id="TVSMainstreet.us" tvg-name="TVS Mainstreet" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/37OGfa3.jpg" group-title="Classic",TVS Mainstreet (264p)
-https://rpn1.bozztv.com/36bay2/gusa-tvsmainst/tracks-v1a1/index-1623401523-now.m3u8
 #EXTINF:-1 tvg-id="TVSMusicNetwork.us" tvg-name="TVS Music Network" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/pXOyNEy.png" group-title="",TVS Music Network
 https://rpn1.bozztv.com/36bay2/gusa-tvsmusic/tracks-v1a1/mono.m3u8
 #EXTINF:-1 tvg-id="TVSNostalgia.us" tvg-name="TVS Nostalgia" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/pXOyNEy.png" group-title="Classic",TVS Nostalgia (720p)
 https://rpn1.bozztv.com/36bay2/gusa-nostalgia/tracks-v1a1/mono.m3u8
 #EXTINF:-1 tvg-id="TVSNostalgiaMovies.us" tvg-name="TVS Nostalgia Movies" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/5Ygj2DL.jpg" group-title="Classic",TVS Nostalgia Movies (480p)
 https://rpn1.bozztv.com/36bay2/gusa-tvsNostalgiaMovies/tracks-v1a1/mono.m3u8
-#EXTINF:-1 tvg-id="TVSPetParadeNetwork.us" tvg-name="TVS Pet Parade Network" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/pXOyNEy.png" group-title="Classic",TVS Pet Parade Network (480p)
-https://rpn1.bozztv.com/36bay2/gusa-petparadenetwork/tracks-v1a1/index-1623401880-now.m3u8
-#EXTINF:-1 tvg-id="TVSPinballNetwork.us" tvg-name="TVS Pinball Network" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/pXOyNEy.png" group-title="Kids",TVS Pinball Network (1080p)
-https://rpn1.bozztv.com/36bay2/gusa-TVSCartoonNetwork/tracks-v1a1/index-1623402005-now.m3u8
-#EXTINF:-1 tvg-id="TVSQuizShowNetwork.us" tvg-name="TVS Quiz Show Network" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/pXOyNEy.png" group-title="Classic",TVS Quiz Show Network (480p)
-https://rpn1.bozztv.com/36bay2/gusa-tvsgameshow/tracks-v1a1/index-1623402120-now.m3u8
```

Comment:
Not quite sure about the name of this one...

Replacement link :  https://rpn1.bozztv.com/36bay2/gusa-tvsgameshow/index.m3u8